### PR TITLE
ZZbbtautau analysis samples

### PR DESCRIPTION
**Pull request by elenavernazza**
Samples needed for the ZZbbtautau validation analysis

### CODE REVIEW

**Comment by kandrosov** [comment]
@elenavernazza please resolve conflicts

File: NanoProd/crab/Run2_2016/TTX.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+TTWJetsToLNu: /TTWJetsToLNu_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1/MINIAODSIM
+TTWJetsToQQ: /TTWJetsToQQ_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1/MINIAODSIM
+TTWW: /TTWW_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1/MINIAODSIM
+TTWZ: /TTWZ_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1/MINIAODSIM
+TTWH: /TTWH_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
+TTZH: /TTZH_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
+TTZToLLNuNu_M-10: /TTZToLLNuNu_M-10_TuneCP5_13TeV-amcatnlo-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1/MINIAODSIM
+TTZToQQ: /TTZToQQ_TuneCP5_13TeV-amcatnlo-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1/MINIAODSIM
+TTZZ: /TTZZ_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1/MINIAODSIM
```
Comment (kandrosov) [inline NanoProd/crab/Run2_2016/TTX.yaml]: Please add TTHH samples for completeness (same for other eras)
```
/TTHHTo2B2Tau_HEFT_c2-3_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
/TTHHTo2B2Tau_HEFT_c2-6_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
/TTHHTo2B2Tau_HEFT_c2-m1_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
/TTHHTo2B2Tau_HEFT_kl-0p5_c2-1_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
/TTHHTo2B2Tau_HEFT_kl-2_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
/TTHHTo2B2Tau_HEFT_kl-3_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
/TTHHTo2B2Tau_HEFT_kt-2_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
/TTHHTo2B2Tau_TuneCP5_13TeV-madgraph-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
```

**Review by kandrosov — CHANGES_REQUESTED** [review]

**Comment by DennRoy** [comment]
Hi, I have a comment and a question:
- Some of the HTauTau samples you're adding in the `Higgs.yaml` files are already present. Can you remove the duplicates?
- You're adding 2016 samples, but don't you need 2016_HIPM samples as well?

**Comment by elenavernazza** [comment]
Hi, thank you for checking!
The changes should be implemented in the last commits, please let me know if any other correction is needed.
Best,

Elena

File: NanoProd/crab/Run2_2016_HIPM/TTX.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+config:
+  params:
+    sampleType: mc
+    era: Run2_2016
```
Comment (DennRoy) [inline NanoProd/crab/Run2_2016_HIPM/TTX.yaml]: `era: Run2_2016_HIPM`

**Review by kandrosov — APPROVED** [review]
@elenavernazza thank you!

**Comment by DennRoy** [comment]
Actually I notice the `/ttHToNonbb_M125_TuneCP5_13TeV-powheg-pythia8/...` dataset are still duplicate in the `Higgs.yaml` files. Could you please remove those too?
Secondly, I wonder if for consistency, the Higgs samples you added should have `_M125` in the names, like the other samples, but I'll leave that to you

**Review by kandrosov — APPROVED** [review]

**Review by DennRoy — APPROVED** [review]
Thanks! I have no other comments

**Pull request merged**